lib/tuya.js

```diff
@@ -1165,6 +1165,11 @@ const tuyaExposes = {
         .withDescription('Result of the self-test'),
     faultAlarm: () => exposes.binary('fault_alarm', ea.STATE, true, false).withDescription('Indicates whether a fault was detected'),
     silence: () => exposes.binary('silence', ea.STATE_SET, true, false).withDescription('Silence the alarm'),
+    frostProtection: () => exposes.binary('frost_protection', ea.STATE_SET, 'ON', 'OFF').withDescription('When Anti-Freezing function'+
+        ' is activated, the temperature in the house is kept at 8 °C, the device display "AF". Press the pair button to cancel.'),
```

Comment:
The CloudEven TRV has no frost protection indication on its display. It can be checked and set in the service menu of the device.
```suggestion
        ' is activated, the temperature in the house is kept at 8 °C.'),
```